app/src/components/main/admin/sidebar/sidebar.stories.jsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { storiesOf } from '@storybook/react';
+import { host } from 'storybook-host';
+import { withReadme } from 'storybook-readme';
+import AdminSidebar from './sidebar';
+import README from './README.md';
+
+storiesOf('Components/Main/Admin/Sidebar', module)
+  .addDecorator(host({
+    title: 'AdminSidebar component',
+    align: 'center middle',
+    backdrop: 'rgba(70, 69, 71, 0.2)',
+    background: '#ffffff',
+    height: 1000,
+    width: '100%',
+  }))
+  .addDecorator(withReadme(README))
+  .add('default state with active settings', () => (
+    <AdminSidebar activePage={'administrate/settings'} />
```

Comment:
1) "default state" means that you didn't pass any props into component. Here "state with with icon" is described.
2) Write more stories (default state, with active page, with actions...)